File: app/src/main/java/com/bearcave/passageplanning/thames_tide_provider/TideDownloaderService.java
package com.bearcave.passageplanning.thames_tide_provider;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;
import android.support.annotation.IntDef;

public class TideDownloaderService extends Service {
    public TideDownloaderService() {
    }

    @Override
    public IBinder onBind(Intent intent) {
        throw new UnsupportedOperationException("Not yet implemented");
    }

    @Override
    public void onCreate() {
        super.onCreate();

    }

    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        return super.onStartCommand(intent, flags, startId);
    }

    @Override
    public boolean onUnbind(Intent intent) {
        return super.onUnbind(intent);
    }
}

File: app/src/main/java/com/bearcave/passageplanning/tasks/TideDownloaderService.kt
package com.bearcave.passageplanning.tasks

import android.app.Service
import android.content.Intent
import android.os.IBinder

class TideDownloaderService : Service() {

    override fun onBind(intent: Intent): IBinder? {
        throw UnsupportedOperationException("Not yet implemented")
    }

    override fun onCreate() {
        super.onCreate()

    }

    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
        return super.onStartCommand(intent, flags, startId)
    }

    override fun onUnbind(intent: Intent): Boolean {
        return super.onUnbind(intent)
    }
}
